config/environments/development.rb

```diff
@@ -115,6 +115,11 @@
   REPOSITORY_STORAGE = "#{::Rails.root.to_s}/data/dev/repos"
 
   ###################################################################
+  # Directory where Repositories will be created. Make sure MarkUs is allowed
+  # to write to this directory
+  REPOSITORY_STORAGE_GITOLITE = "#{::Rails.root.to_s}/data/dev/repos/gitoliteRepos"
```

Comment:
Hmmm. I'm not really sure `to_s` is more human-readable, but I don't care. You can leave it.